depends/README.md

```diff
@@ -49,6 +49,10 @@ The paths are automatically configured and no other options are needed unless ta
 
     sudo apt-get install curl librsvg2-bin libtiff-tools bsdmainutils cmake imagemagick libz-dev python3-setuptools libtinfo5 xorriso
 
+Note: You must obtain the macOS SDK before proceeding with a cross-compile.
+The SDK should be placed under `depends/SDKs/`.
+For more information: [SDK Extraction](../contrib/macdeploy/README.md#sdk-extraction)
```

Comment:
nit, I think the presentation with colon and no period at the end would be good if the URL is displayed, or in this case, the rendering might be better with:
```suggestion
For more information, see [SDK Extraction](../contrib/macdeploy/README.md#sdk-extraction).
```

Reply:
will do, pending your response to: https://github.com/bitcoin/bitcoin/pull/21688#discussion_r615428634

Reply:
addressed in 5e7d1997fce15b3e2fb77e67491237812e1f98d9